client/js/upload.ts

```diff
@@ -11,12 +11,18 @@ class Uploader {
 	overlay: HTMLDivElement | null = null;
 	uploadProgressbar: HTMLSpanElement | null = null;
 
+	onDragEnter = (e: DragEvent) => this.dragEnter(e);
```

Comment:
we have to pass a reference as the second arg to removeEventListener for it to work properly.